app/uk/gov/hmrc/nisp/views/excluded_cope_failed.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+@import uk.gov.hmrc.nisp.views.viewParams.MainTemplateParams
+@import uk.gov.hmrc.nisp.views.html.unauthenticatedMain
+@import uk.gov.hmrc.nisp.controllers.auth.ExcludedAuthenticatedRequest
+@import uk.gov.hmrc.nisp.models.Exclusion
+@import uk.gov.hmrc.nisp.models.Exclusion._
+@import uk.gov.hmrc.play.partials._
+@import uk.gov.hmrc.renderer._
+@import uk.gov.hmrc.nisp.config.ApplicationConfig
+
+@this(appConfig: ApplicationConfig)
+
+@(exclusion: Exclusion, statePensionAge: Option[Int])(implicit messages: Messages, request: ExcludedAuthenticatedRequest[_], partialRetriever: CachedStaticHtmlPartialRetriever,
+formPartialRetriever: FormPartialRetriever, templateRenderer: TemplateRenderer)
+
+@analyticsAdditionalJs = @{
+    Map[String, Any](
+        "nispExclusionType" -> exclusion.toString,
+        "statePensionAge" -> statePensionAge,
+        "authProvider" -> request.authDetails.authProvider,
+        "confidenceLevel" -> request.authDetails.confidenceLevel
+    )
+}
+
+@excludeUrPanel(showUrBannerOn: Exclusion) = @{
+    showUrBannerOn == PostStatePensionAge | showUrBannerOn == IsleOfMan
+}
+
+@unauthenticatedMain(MainTemplateParams(userLoggedIn = true, browserTitle = Some(messages("nisp.main.title")),
+            pageTitle = Some(messages("nisp.excluded.cope.failed.h1")), articleClasses = Some("mainpage"))
+) {
+    <div>
+        <p>@Html(messages("nisp.excluded.cope.processing.failed"))</p>
+
+        <a id="futurePensionLink" href="@appConfig.futurePensionUrl">@Html(messages("nisp.excluded.cope.processing.failed.link"))</a>
```

Comment:
There looks like a lot more content on the screenshot than what is here

Reply:
Updated the pages to match revised designs/content.